compiler/check-artifact.sh

```diff
@@ -108,6 +110,8 @@ checkDependencies ()
     elif [[ "$ARCH" == aarch_64 ]]; then
       dump_cmd='aarch64-linux-gnu-objdump -x '"$1"' |grep "NEEDED"'
       white_list="linux-vdso\.so\.1\|libpthread\.so\.0\|libm\.so\.6\|libc\.so\.6\|ld-linux-aarch64\.so\.1"
+    elif [[ "$ARCH" == ppcle_64 ]]; then
+      white_list="linux-vdso64\.so\.1\|libpthread\.so\.0\|libm\.so\.6\|libc\.so\.6\|ld64\.so\.2\|libstdc++\.so\.6\|libgcc_s\.so\.1"
```

Comment:
I don't think libstdc and libgcc_s should be in this list. If those show up, then the static linking isn't working as we'd hope.

I don't think the x86 ldd can process the ppc binary here. It seems dump_cmd would need to be defined.